mindsdb_sdk/model.py

```diff
@@ -51,39 +51,51 @@ def predict(self, data: Union[pd.DataFrame, Query], params: dict = None) -> pd.D
         if isinstance(data, Query):
             # create join from select if it is simple select
             ast_query = parse_sql(data.sql, dialect='mindsdb')
-            if isinstance(ast_query, Select) and isinstance(ast_query.from_table, Identifier):
-                # inject aliases
-                if ast_query.from_table.alias is None:
-                    alias = 't'
-                    ast_query.from_table.alias = Identifier(alias)
-                else:
-                    alias = ast_query.from_table.alias.parts[-1]
-
-                def inject_alias(node, is_table, **kwargs):
-                    if not is_table:
-                        if isinstance(node, Identifier):
-                            if node.parts[0] != alias:
-                                node.parts.insert(0, alias)
-
-                query_traversal(ast_query, inject_alias)
-
-                # replace table with join
-                ast_query.from_table = Join(
+
+            # injection of join disabled yet
```

Comment:
I left it because now I'm not 100% sure that we won't use this mode. When this SDK will become well used it starts to be more complex and maybe this mode will be use for some use case. 
I left it not hidden in commit history to not forget about it.